actors/builtin/paych/paych_actor.go

```diff
@@ -154,10 +156,14 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 		rt.Abortf(exitcode.ErrIllegalArgument, "voucher signature invalid")
 	}
 
-	if rt.CurrEpoch() < sv.TimeLock {
+	if rt.CurrEpoch() < sv.TimeLockMin {
 		rt.Abortf(exitcode.ErrIllegalArgument, "cannot use this voucher yet!")
 	}
 
+	if rt.CurrEpoch() > sv.TimeLockMax {
```

Comment:
We should check that the TimeLockMax is not zero, to allow people to set it to zero and mean 'no timeout'

Reply:
Good point, thanks!